lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -506,16 +549,18 @@ def get_students_problem_grades(request, csv=False):
         # Adding 1 so can tell if list is larger than MAX_SCREEN_LIST_LENGTH
         # without doing another select.
         for student in students[0:MAX_SCREEN_LIST_LENGTH + 1]:
```

Comment:
This bit of jiggery pokery is for ensuring that a vertical scroll bar gets added to the window if the list of students is too long. I believe your addition will mess up the counting for this.